tests/core/test_scan.py

```diff
@@ -34,3 +48,191 @@ def test_unique_identifiers(unique_id_mock, response):
     assert "id2" == next(identifiers)
     unique_id_mock.assert_called_with("_service2._tcp.local", "service2", {"c": "d"})
     assert not next(identifiers, None)
+
+
+@pytest.mark.asyncio
+async def test_scan_with_zeroconf():
+    if not hasattr(asyncio, "get_running_loop"):
+        # until python 3.6 is removed, its EOL anyways
+        asyncio.get_running_loop = asyncio.get_event_loop
```

Comment:
This needs to get removed, but I didn't want to assume its ok to drop python3.6 support

zeroconf no longer supports 3.6 since it doesn't support EOL py3 versions

Reply:
I will deprecate python 3.6 according to #1587. Hopefully I can do that today.

Reply:
#1587 is merged now.